internal/account/idgenerator.go

```diff
@@ -0,0 +1,24 @@
+package account
+
+import (
+	"crypto/rand"
+	"math/big"
+	"strings"
+)
+
+const CHARSET = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
I dont think we should name this CHARSET, since capitalizing it would mean exposing it globally
```suggestion
const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Reply:
Nice catch, thanks!